### DISCUSSION TITLE
SSL error

### DISCUSSION THREAD

**Discussion by mrkobz** [emersonfelipesp/netbox-proxbox#67] [General]
After integrate proxbox i have error ->
Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate
How can I skip certificate verification?

**Comment by emersonfelipesp** [comment]
Could you please update NetBox to v3.5.2 and Proxbox to latest code? I think it will work.
